utils/oscap-tool.c

```diff
@@ -35,6 +35,7 @@
 #include <errno.h>
 #include <limits.h>
 #include <cvss_score.h>
+#include "../src/common/debug.h"
```

Comment:
This needs to be changed to `#include <oscap_debug.h>`

The `oscap` tools is compiled only using public headers. So, when we introduce a new dependency we need to make it public. That way we ensure that anyone else (without the library sources can implement their own scanner).

Please move `debug.h` to `public/oscap_debug.h` and also update the `src/common/Makefile.am` appropriately.